web/push-notif/notif-crypto-utils.js

```diff
@@ -52,6 +53,7 @@ export type WebNotifDecryptionError = {
 
 export type WebNotifsServiceUtilsData = {
   +olmWasmPath: string,
+  +vodozemacWasmPath: string,
```

Comment:
Shouldn't this be optional, since it's persisted and the current version won't have `vodozemacWasmPath`?